docs/sources/userguide/dockervolumes.md

```diff
@@ -24,7 +24,9 @@ containers that bypasses the [*Union File
 System*](/terms/layer/#union-file-system) to provide several useful features for
 persistent or shared data:
 
-- Volumes are initialized when a container is created
+- Volumes are initialized when a container is created. Any data in the image
```

Comment:
The bullet is a bit awkward to parse ...maybe this
--------
## Data volumes

A *data volume* is a specially-designated directory within one or more
containers that bypasses the [*Union File
System*](/terms/layer/#union-file-system). Data volumes provide several useful features for persistent or shared data:

- Volumes are initialized when a container is created. If the container's base image contains data at the specified mount point, that data is copied into the new volume.
- Data volumes can be shared and reused among containers.
- Changes to a data volume are made directly.
- Changes to a data volume are not be included when you update an image.
- Data volumes persist even if the container itself is deleted.

Data volumes are designed to persist data, independent of the container's life 
cycle. Docker therefore *never* automatically deletes volumes when you remove 
a container, nor does it "garbage collect" volumes the container no longer references.